src/Wizard/WizardSidebar.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+
+import { ListGroup, ListGroupItem } from '../index.js'
+
+/**
+ * WizardSidebar component for Patternfly React
+ */
+export const WizardSidebar = ({ items, className, ...rest }) => {
+  const classes = cx('wizard-pf-sidebar', className)
+  return (
+    <div className={classes} {...rest}>
+      {items}
+    </div>
+  )
+}
+WizardSidebar.propTypes = {
+  /** Wizard sidebar items */
+  items: PropTypes.node,
+  /** Additional css classes */
+  className: PropTypes.string
+}
+
+/**
+ * WizardSidebarGroup component for Patternfly React
+ */
+export const WizardSidebarGroup = ({
+  children,
+  className,
+  step,
+  activeStep,
+  ...rest
+}) => {
+  const classes = cx({ hidden: step !== activeStep }, className)
+  return (
+    <ListGroup componentClass="ul" className={classes} {...rest}>
+      {children}
+    </ListGroup>
+  )
+}
+WizardSidebarGroup.propTypes = {
+  /** Children nodes */
+  children: PropTypes.node,
+  /** Additional css classes */
+  className: PropTypes.string,
+  /** The wizard step number for this step */
+  step: PropTypes.string,
+  /** The active step */
+  activeStep: PropTypes.string
+}
+
+/**
+ * WizardSidebarGroupItem component for Patternfly React
+ */
+export const WizardSidebarGroupItem = ({
+  stepIndex,
+  subStepIndex,
+  className,
+  subStep,
+  label,
+  title,
+  activeSubStep,
+  onClick,
+  ...rest
+}) => {
+  const classes = cx({ active: subStep === activeSubStep }, className)
+  return (
+    <ListGroupItem className={classes} listItem {...rest}>
+      <a
+        href="#"
```

Comment:
+1

Reply:
actually...I believe this is OK as long as the event is handled, i.e. `preventDefault()`

Reply:
In OpenShift, they are moving towards replacing anchors with buttons styled with btn-link to provide better accessibility. Is that something we want to consider here (and in general)?

Reply:
@jeff-phillips-18 i think both are acceptable from what I've seen before. Do you have an example some place? I can try to mimic the anchor with a button and see how it goes...I noted that the ListGroupItem would support buttons.

Reply:
also - do we have any a11y standards on these sort of things any place yet? I've been asking, but can't seem to find them. I've been using the PF Core test pages as best source of truth, but will turn elsewhere and start making notes...

Reply:
Aside from saying that semantic html is accessible for free, I'm not aware of any standards that we have. Nor am I aware of what screen readers we should support. For now, I think the best thing we can do is to test using the devices we have access to.
Here is an article in favor of using `<button>` since accessibility is enabled by default: https://docs.google.com/spreadsheets/d/1PEtJX8LpzXvs8WicUaq61NnVaQpgeTBMgXtxJd2HslQ/edit#gid=1608797239
The article also provides details on how to enable an `<a>` element to be accessible using additional javascript if we are tied to using `<a>`.

Reply:
I believe the ideas is to not require the call to preventDefault and to show the focus border when the item has focus.

Reply:
I will try the button and see if it impacts styling within the current PF styles for this component...

Reply:
Ok a quick browser test shows that Buttons do not have styles here.. i.e.: we have styles for:
`.wizard-pf-sidebar .list-group-item>a` but not: `.wizard-pf-sidebar .list-group-item> button` it seems. 

So are we OK w/ using an `<a>`, or should we halt until pushing some additional CSS?

Reply:
Yes, I would continue with using <a> for now. PF core is currently working on a new wizard, so we can switch to that when it's available.

The [article](https://www.deque.com/blog/accessible-aria-buttons/) I mentioned also talks about what is needed when using `<a>` instead of `<button>`, such as using .preventDefault() (although that may not be needed if we using `href=""` without the `#`?).

(and I fixed the link to that article that I pasted in my previous comment)
![bitmoji](https://render.bitstrips.com/v2/cpanel/1ca29674-6b9a-4b0f-8665-6197c31b7fd6-a07a3b9e-da41-4f2c-af4f-5c7d9c4da655-v1.png?transparent=1&palette=1&width=123)